app/client/stylesheets/_application.scss

```diff
@@ -0,0 +1,14 @@
+.error-message{
+  color: red;
+}
+
+.essay-error-flexcontainer{
```

Comment:
see above, also rename `.essay-error-flexcontainer` to `.essay-feedback-container`